inductiva/utils/format_utils.py

```diff
@@ -113,4 +147,28 @@ def get_tabular_str(tabular_data: Union[Mapping[str, Iterable[Any]],
     """
 
     data, headers = get_tabular_data(tabular_data, headers, formatters)
-    return tabulate(data, headers=headers, missingval="n/a")
+
+    header_formatters = header_formatters or []
+
+    for formatter in header_formatters:
+        headers = [formatter(header) for header in headers]
+
+    # pylint: disable=protected-access
+    tabulate._table_formats["inductiva"] = TableFormat(
+        lineabove=None,
+        linebelowheader=None,
+        linebetweenrows=None,
+        linebelow=None,
+        headerrow=DataRow("", " ", ""),
+        datarow=DataRow("", " ", ""),
+        padding=3,
+        with_header_hide=None,
+    )
```

Comment:
This should be a constant on top.

Reply:
Not even a constant. This just needs to be added once when we import the lib.
Do something like:

```python
<imports>
import tabulate

tabulate._table_formats["inductiva"] = TableFormat(
        lineabove=None,
        linebelowheader=None,
        linebetweenrows=None,
        linebelow=None,
        headerrow=DataRow("", " ", ""),
        datarow=DataRow("", " ", ""),
        padding=3,
        with_header_hide=None,
    )

<rest of the code here>
```